proc/sort.go

```diff
@@ -387,12 +387,15 @@ func (r *runFile) read() (*zng.Record, bool, error) {
 // writeZng writes records to w as a zng stream.
 func writeZng(w io.Writer, records []*zng.Record) error {
 	bw := bufio.NewWriter(w)
-	zw := zngio.NewWriter(bw, zio.WriterFlags{})
+	zw := zngio.NewWriter(bw, zio.WriterFlags{ZngLZ4BlockSize: zio.DefaultZngLZ4BlockSize})
```

Comment:
I don't think we should compress the temporary spill files. I ran a single battery of tests locally, using a 9GB conn.log (`wrccdc-year1/zeek-logs/conn.log`) on my laptop, executing `zq "* | sort -r ts"` & verifying spilling was occurring.

```
no compression at all (using master branch, not this PR's branch)
real	7m32.483s
user	9m41.012s
sys	0m36.228s


compression for sort & output
real	7m49.170s
user	10m30.612s
sys	0m24.446s


compression for output, not for sort spills
real	7m33.253s
user	10m4.455s
sys	0m33.602s
```

Resulting in these files:
```
08:07 $ ls -lh *zng
-rw-r--r--  1 alfred  staff   3.2G Jul 31 08:00 comp-no-sort-comp.zng
-rw-r--r--  1 alfred  staff   3.2G Jul 31 07:39 comp.zng
-rw-r--r--  1 alfred  staff   6.1G Jul 31 07:48 noncomp.zng
```

Reply:
Disabled in 850f0e5c73699008a87c8cba592ba91b3ad4bdc8.